### ISSUE TITLE
a: visited link text (purple) FAILS colour contrast when background is dark blue, such as narrative pane, feedback overlay etc.

### ISSUE THREAD

**Issue by graphicidentity** [MeD-DMC/adapt-esdc-college-theme#73]
Will provide more examples. Please stay tuned...

**Comment by ftourangeau** [comment]
fixed in 1.6.7
